indigo_api/models/works.py

```diff
@@ -638,6 +648,29 @@ def versions(self):
     def as_at_date(self):
         return self.as_at_date_override or self.place.settings.as_at_date
 
+    def approve(self, user, request=None):
+        try:
+            with transaction.atomic():
+                self.work_in_progress = False
+                self.approved_by_user = user
+                self.approved_at = datetime.now()
+                self.save_with_revision(user)
+                work_approved.send(sender=self, request=request)
```

Comment:
the sender is (weirdly) the class, and instance is self

```suggestion
                work_approved.send(sender=self.__class__, instance=self, request=request)
```

Reply:
I don't think we should pass in the request since we won't always have it.